README.md

```diff
@@ -331,6 +331,19 @@ eksctl apply --cluster-config advanced-cluster.yaml
 ```
 -->
 
+### Shell Completion
+
+To enable bash completion, run the following, or put it in `~/.bashrc` or `~/.profile`:
+```
+. <(eksctl completion bash)
+```
+
+Or for zsh:
+```
+. <(eksctl completion zsh)
```

Comment:
i'm also fairly new to zsh completion but i just tried it and i don't think that's how it's going to work unfortunately.

My brief research has led me to something like this

```
mkdir ~/.zsh/completion/
eksctl completion zsh > ~/.zsh/completion/_eksctl
```
Then add this to the `fpath` in `~/.zshrc`
```
 fpath=(~/.zsh/completion $fpath)
```